xdsl/dialects/test.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+from xdsl.ir import Attribute, Data, Dialect, MLIRType, OpResult, Operation
+
+from xdsl.irdl import VarOpResult, irdl_attr_definition, irdl_op_definition
+from xdsl.parser import BaseParser
+from xdsl.printer import Printer
+
+
+@irdl_op_definition
+class ProduceValuesOp(Operation):
```

Comment:
Can we add a comment here to explain how this is meant to be used?
That should make it clear for new users.